src/Wavefront.AspNetCore.SDK.CSharp/Common/WavefrontAspNetCoreReporter.cs

```diff
@@ -40,13 +42,23 @@ public class WavefrontAspNetCoreReporter
         /// <value>The source/host name.</value>
         public string Source { get; }
 
+        private WavefrontSdkMetricsRegistry sdkMetricsRegistry;
+
         private WavefrontAspNetCoreReporter(IMetricsRoot metrics, IWavefrontSender wavefrontSender,
                                             ApplicationTags applicationTags, string source)
         {
             Metrics = metrics;
             WavefrontSender = wavefrontSender;
             ApplicationTags = applicationTags;
             Source = source;
+            sdkMetricsRegistry = new WavefrontSdkMetricsRegistry
+                .Builder(wavefrontSender)
+                .Prefix(SdkMetricPrefix + ".asp_net")
```

Comment:
Let's include `core` in the prefix as well since this SDK is exclusively targeting .NET Core, so perhaps `.aspnetcore` instead